lib/stacks/TemplateStack.ts

```diff
@@ -4,6 +4,9 @@ import TemplateConstruct from "../constructs/TemplateConstruct";
 export default class Template extends Stack {
   constructor(scope: Construct, id: string, props?: StackProps) {
     super(scope, id, props);
-    new TemplateConstruct(this, id, { });
+    new TemplateConstruct(this, id, { 
+      vpcId: 'vpc-0a2cad50c98aed83f'    // optional param to use existing VPC; if not specified, will use default VPC.
```

Comment:
Probably shouldn't be hardcoding a vpc here.

Reply:
Resolved - put in generic "vpc-xxxx" and commented out the line in commit 10c7aaa.